src/Console/Command/CommandHelp.php

```diff
@@ -269,6 +270,21 @@ public static function toString($value)
         return var_export($value, true);
     }
 
+    public static function getDisplayableAllowedValues(FixerOption $option)
+    {
+        if (null !== $allowed = $option->getAllowedValues()) {
```

Comment:
Yep, I only do this in case I have both assignment **and** comparison. In `if ('foo' === $bar = baz()) { }` there is no ambiguity.